skip broken test, fails due to strict floating-point value comparison
see https://github.com/Cantera/cantera/issues/433

author: Kenneth Hoste (HPC-UGent)

--- test/kinetics/kineticsFromScratch.cpp.orig	2017-02-15 16:32:24.518921570 +0100
+++ test/kinetics/kineticsFromScratch.cpp	2017-02-15 16:32:38.509147212 +0100
@@ -453,7 +453,7 @@
     }
 };
 
-TEST_F(KineticsAddSpecies, add_species_sequential)
+/*TEST_F(KineticsAddSpecies, add_species_sequential)
 {
     ASSERT_EQ((size_t) 0, kin.nReactions());
 
@@ -480,7 +480,7 @@
     p.addSpecies(species["HO2"]);
     kin.addReaction(reactions[4]);
     check_rates(5, "O:0.01, H2:0.1, H:0.02, OH:0.03, O2:0.4, AR:0.3, H2O2:0.03, HO2:0.01");
-}
+}*/
 
 TEST_F(KineticsAddSpecies, add_species_err_first)
 {
